assets/styl/_theme/_skin.styl

```diff
@@ -161,6 +179,12 @@
             &:before
                 background-image _inlineSvg('<svg viewBox="0 0 32 32" width="32" height="32"><path fill="#fff" d="M16 0c-8.837 0-16 7.163-16 16s7.163 16 16 16 16-7.163 16-16-7.163-16-16-16zM16 29c-7.18 0-13-5.82-13-13s5.82-13 13-13 13 5.82 13 13-5.82 13-13 13zM12 9l12 7-12 7z"></path></svg>')
 
+    &-podcast .media
+        > p:first-child
+        > div:first-child
+            &:before
+                background-image _inlineSvg('<svg viewBox="0 0 32 32" width="32" height="32"><path fill="#fff" d="M16 0C7.2 0 0 7.2 0 16s7.2 16 16 16 16-7.2 16-16S24.8 0 16 0zm0 29C8.8 29 3 23.2 3 16S8.8 3 16 3s13 5.8 13 13-5.8 13-13 13z"/><path fill="#fff" d="M16 9c-.4 0-.8.3-.8.8v12.5c0 .4.3.8.8.8.4 0 .8-.3.8-.8V9.7c0-.4-.4-.7-.8-.7zM18.4 11.4c-.4 0-.8.3-.8.8v7.6c0 .4.3.8.8.8s.8-.3.8-.8v-7.6c-.1-.4-.4-.8-.8-.8zM20.7 12.9c-.4 0-.8.3-.8.8v4.6c0 .4.3.8.8.8s.8-.3.8-.8v-4.6c0-.4-.4-.8-.8-.8zM23 11c-.4 0-.8.3-.8.8v8.5c0 .4.3.8.8.8s.8-.3.8-.8v-8.5c0-.5-.3-.8-.8-.8zM25.4 13.9c-.4 0-.8.3-.8.8v2.6c0 .4.3.8.8.8.4 0 .8-.3.8-.8v-2.6c0-.4-.4-.8-.8-.8zM13.6 7.9c-.4 0-.8.3-.8.8v14.7c0 .4.3.8.8.8.4 0 .8-.3.8-.8V8.7c0-.5-.3-.8-.8-.8zM11.3 10.6c-.4 0-.8.3-.8.8v9.2c0 .4.3.8.8.8.4 0 .8-.3.8-.8v-9.2c0-.5-.3-.8-.8-.8zM9 13.9c-.4 0-.8.3-.8.8v2.6c0 .4.3.8.8.8.4 0 .8-.3.8-.8v-2.6c-.1-.4-.4-.8-.8-.8zM6.6 13.5c-.4 0-.8.3-.8.8v3.4c0 .4.3.8.8.8s.8-.3.8-.8v-3.4c0-.5-.4-.8-.8-.8z"/></svg>')
```

Comment:
We souldn't use `_inlineSvg` anymore to prefer `svgImport` https://github.com/20minutes/colette/blob/master/assets/styl/_modules/_media.styl#L21